bundles/org.openhab.ui/web/src/components/thing/thing-mixin.js

```diff
@@ -11,7 +11,7 @@ export default {
      */
     validateThingUID (uid, id) {
       if (!/^[A-Za-z0-9_-]+$/.test(id)) {
-        return 'Required. Must not start with a number. A-Z,a-z,0-9,_ only'
+        return 'Required. Must not start with a number. A-Z,a-z,0-9,_,- only'
```

Comment:
I believe it should also not start with dash.

Reply:
Core allows it though.

Reply:
My mistake! It isn't allowed in .things file. I'll adjust.